apps/src/blockly/themes/cdoTheme.js

```diff
@@ -1,7 +1,8 @@
 import GoogleBlockly from 'blockly/core';
```

Comment:
Should this now be called `cdoLight` (vs. `cdoDark`)?

Reply:
I think keeping it as `cdoTheme` may better suggest that it's the default than `cdoLight` which feels more like a specialization. This isn't a super strong opinion. Thoughts?

Reply:
Yeah, that's fair.  If MusicLab is going to be a dark-mode only presentation, we also might need for that one lab to override each theme's component styles. But I think we can cross that bridge when we come to it.

I'm planning to add the other themes soon, so let's table this one potential rename until then?